test/__snapshots__/katex-spec.js.snap

```diff
@@ -1,23 +1,5 @@
-
-exports[`An HTML font tree-builder should render \\textsf{\\textbf{$\\mathrm{A}$}} with the correct font 1`] = `"<span class=\\"katex\\"><span class=\\"katex-mathml\\"><math><semantics><mrow><mstyle scriptlevel=\\"0\\" displaystyle=\\"false\\"><mrow><mi mathvariant=\\"normal\\">A</mi></mrow></mstyle></mrow><annotation encoding=\\"application/x-tex\\">\\\\textsf{\\\\textbf{$\\\\mathrm{A}$}}</annotation></semantics></math></span><span class=\\"katex-html\\" aria-hidden=\\"true\\"><span class=\\"strut\\" style=\\"height:0.68333em;\\"></span><span class=\\"strut bottom\\" style=\\"height:0.68333em;vertical-align:0em;\\"></span><span class=\\"base\\"><span class=\\"mord text\\"><span class=\\"mord text\\"><span class=\\"mord\\"><span class=\\"mord mathrm\\">A</span></span></span></span></span></span></span>"`;
-
-exports[`An HTML font tree-builder should render \\textsf{\\textit{R}G\\textbf{B}} with the correct font 1`] = `"<span class=\\"katex\\"><span class=\\"katex-mathml\\"><math><semantics><mrow><mtext>RGB</mtext></mrow><annotation encoding=\\"application/x-tex\\">\\\\textsf{\\\\textit{R}G\\\\textbf{B}}</annotation></semantics></math></span><span class=\\"katex-html\\" aria-hidden=\\"true\\"><span class=\\"strut\\" style=\\"height:0.69444em;\\"></span><span class=\\"strut bottom\\" style=\\"height:0.69444em;vertical-align:0em;\\"></span><span class=\\"base\\"><span class=\\"mord text\\"><span class=\\"mord text\\"><span class=\\"mord textsf textit\\">R</span></span><span class=\\"mord textsf\\">G</span><span class=\\"mord text\\"><span class=\\"mord textsf textbf\\">B</span></span></span></span></span></span>"`;
-
-exports[`An HTML font tree-builder should render \\textsf{R} with the correct font 1`] = `"<span class=\\"katex\\"><span class=\\"katex-mathml\\"><math><semantics><mrow><mtext>R</mtext></mrow><annotation encoding=\\"application/x-tex\\">\\\\textsf{R}</annotation></semantics></math></span><span class=\\"katex-html\\" aria-hidden=\\"true\\"><span class=\\"strut\\" style=\\"height:0.69444em;\\"></span><span class=\\"strut bottom\\" style=\\"height:0.69444em;vertical-align:0em;\\"></span><span class=\\"base\\"><span class=\\"mord text\\"><span class=\\"mord textsf\\">R</span></span></span></span></span>"`;
-
```

Comment:
I totally get why you reverted these.  Thanks for trying.  I'll try to make the snapshots better in the future.  The tests you have are good for now.